docs/data/pages.ts

```diff
@@ -296,123 +298,7 @@ const pages: MuiPage[] = [
       {
         pathname: '/x/api/date-pickers-group',
         title: 'API Reference',
-        children: [
```

Comment:
Yes, I thought it would be mixed with interface API pages, but it's not the case 👍

I spent some time trying to do the same with interfaces, but it does not worths it, because the menu is not alphabetically sorted. It's manualy organized.

For example `GridExcelExportOptions`, `GridPrintExportOptions`, `GridCsvExportOptions` should stay one after the other which is not the case for alphabetical order.